spotirec.py

```diff
@@ -398,6 +419,55 @@ def print_blacklist():
             print('Blacklist is empty')
 
 
+def print_user_genres_sorted(prompt=True):
+    """
+    Print user top genres to terminal in a formatted list.
+    :param prompt: whether or not user should be prompted for input
+    """
+    sort = sorted(get_user_top_genres().items(), key=lambda kv: kv[1], reverse=True)
+    print_choices([sort[x][0] for x in range(0, len(sort))], prompt=prompt)
+
+
+def get_genre_seeds() -> json:
+    """
+    Retrieves available genre seeds from Spotify API.
+    :return: genre seeds as a json obj
+    """
+    response = requests.get(f'{url_base}/recommendations/available-genre-seeds', headers=headers)
+    return json.loads(response.content.decode('utf-8'))
```

Comment:
Unnecessary variable declaration. This can be reduced to a single line.

Reply:
Not quite. We need the response to implement error handling with status codes per #9

Reply:
Ah, okay then